generated_code/aderdg.py

```diff
@@ -125,13 +125,11 @@ def __init__(self, order, multipleSimulations, matricesDir):
     self.db.update(project2nFaceTo3m)
 
     selectVelocitySpp = np.zeros((self.numberOfQuantities(), 3))
```

Comment:
You could e.g. take `extractVelocities` here instead and transpose the resulting matrix—i.e. the `mapToVelocities` method.